src/components/chat/ChatEntryTemplate.vue

```diff
@@ -8,9 +8,13 @@
       }"
       v-if="!brief"
     >
-      <div v-if="amount > 0 || direction === 'from'" class="message-tick" :data-confirmation="confirm"></div>
+      <div v-if="amount > 0 || direction === 'from'" class="message-tick" :data-confirmation="confirm">
+        <md-icon>{{ messageTick[confirm] }}</md-icon>
```

Comment:
Icon duplication https://snag.gy/Z2NVQz.jpg

Reply:
Icons in fixed again:
https://github.com/Adamant-im/adamant-im/pull/210/commits/0725cafc901c928028fc1d16695900e0cae5a3b3
New pseudo element icons appeared in #205:
https://github.com/Adamant-im/adamant-im/pull/205/files#diff-62b2e445c95b8025c7698fb8ce231de8R133

Reply:
https://github.com/Adamant-im/adamant-im/pull/210/commits/aa81a37c4fcc360460e3381ad41294f56665fef7